test/acceptance/behave/components/ngsiv2/subscriptions/notifications/notifications_metadata.feature

```diff
@@ -0,0 +1,493 @@
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# author: 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+#
+#  Note: the "skip" tag is to skip the scenarios that still are not developed or failed
+#        -tg=-skip
+#
+
+
+Feature: verify notifications with specials metadata from subscriptions using NGSIv2
+  As a context broker user
+  I would like to verify notifications with specials metadata from subscriptions using NGSIv2
+  So that I can manage and use them in my scripts
+
+  Actions Before the Feature:
+  Setup: update properties test file from "epg_contextBroker.txt" and sudo local "false"
+  Setup: update contextBroker config file
+  Setup: start ContextBroker
+  Check: verify contextBroker is installed successfully
+  Check: verify mongo is installed successfully
+  Setup: start the subscription listener as a daemon using the port "1045"
+
+  Actions before each Scenario:
+  Check: verify subscription listener is started successfully
+
+  Actions After each Scenario:
+  Setup: delete database in mongo
+
+  Actions After the Feature:
+  Setup: stop ContextBroker
+
+
+  # ----------------------- metadata  ---------------------------
+  @metadata_without
+  Scenario:  send a notification using NGSI v2 without special metadata
+    Given  a definition of headers
+      | parameter          | value               |
+      | Fiware-Service     | test_notif_metadata |
+      | Fiware-ServicePath | /test               |
+      | Content-Type       | application/json    |
+    # These properties below are used in subscriptions request
+    And properties to subscriptions
+      | parameter             | value                           |
+      | subject_idPattern     | .*                              |
+      | condition_attrs       | without condition field         |
+      | notification_http_url | http://replace_host:1045/notify |
+    And create a new subscription
+    And verify that receive a "Created" http code
+    # These properties below are used in entity request
+    And properties to entities
+      | parameter         | value       |
+      | entities_type     | random=3    |
+      | entities_id       | room1       |
+      | attributes_number | 3           |
+      | attributes_name   | temperature |
+      | attributes_value  | random=5    |
+      | attributes_type   | celsius     |
+      | metadatas_number  | 2           |
+      | metadatas_name    | very_hot    |
+      | metadatas_type    | alarm       |
+      | metadatas_value   | hot         |
+    When create entity group with "1" entities in "normalized" mode
+    And verify that receive several "Created" http code
+    Then get notification sent to listener
+    And verify metadata in notification with "*"
```

Comment:
In particular, what the meaning of "*" is (when `<metadata>` is used, I think I have got the point...)